src/song/QUSongFile.h

```diff
@@ -59,7 +59,7 @@ class QUSongFile: public QUSongInterface {
 	// comparing functions
 	static bool equal(QUSongFile *s1, QUSongFile *s2);
 
-public slots:
+public:

```

Comment:
I get a warning on the console when starting USM, which I assume was due to this change:
`qt.core.qobject.connect: QObject::connect: No such slot QUSongFile::songFileChanged(const QString&)`
I don't know enough Qt to understand what this change does though.

Reply:
I trusted clazy on that, but I'd rather take that change back.

Reply:
Could you check that the error is gone now after the additional fixes?

Reply:
Looks good now